### PR TITLE
Adobe workflow

### PR DESCRIPTION
**Pull request by tissuematter**
link to psd file in readme. png in catalog.

### CODE REVIEW

**Comment by tissuematter** [comment]
quicktime screen capture of process https://youtu.be/vGIAlmU3EnE

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Ignore Mac DS_Store files
```
Comment (mrpaws) [inline .gitignore:1]: Thank you for using comments!

**Review by mrpaws — APPROVED** [review]
Great documentation and discussion in the code base here, Jon.  Really appreciate the help so far.  The collage looks AMAZING.

**Comment by mrpaws** [comment]
Great documentation and discussion in the code base here, Jon.  Really appreciate the help so far.  The collage looks AMAZING.

**Pull request merged**